db/re/item_db.conf

```diff
@@ -39438,6 +39479,20 @@ item_db: (
 	Buy: 20
 	Weight: 10
 	Loc: 2
+	Script: <"
+		bonus2 bAddDamageClass,2085,30;
+		bonus2 bAddDamageClass,2090,30;
+		bonus2 bAddDamageClass,2144,30;
+		bonus2 bAddDamageClass,2145,30;
+		bonus2 bAddDamageClass,2083,30;
+		bonus2 bAddDamageClass,2088,30;
+		bonus2 bAddDamageClass,2084,30;
```

Comment:
I think, we don't have AddDamageClass bonus
All Class bonus are not there, we deal with races.